src/libYARP_sig/include/yarp/sig/Image.h

```diff
@@ -139,7 +172,7 @@ class YARP_sig_API yarp::sig::Image : public yarp::os::Portable {
      */
     inline int getPadding() const
     {
-        const int ret=imgRowSize-imgWidth*imgPixelSize;
+        const int ret=imgRowSize-imgWidth*getPixelSize();
```

Comment:
is there a reason for not using `imgPixelSize` here?

Reply:
see the replies to similar questions below